lib_eio_linux/low_level.ml

```diff
@@ -524,6 +524,15 @@ let read_link fd path =
     Eio_unix.run_in_systhread ~label:"read_link" (fun () -> Eio_unix.Private.read_link (Some parent) leaf)
   with Unix.Unix_error (code, name, arg) -> raise @@ Err.wrap_fs code name arg
 
+let chown ~follow ~uid ~gid fd path =
+  let module At = Uring.Linkat_flags in
+  let follow = if follow then At.empty_path else At.(empty_path + symlink_follow) in
```

Comment:
This looks wrong. The flag is `nofollow`, not `follow`.

Reply:
The Docker container has groups `opam` and `users`, so we could test it. Maybe just get the list of groups the current process is in and try switching the file to a different one?